src/Core/AutoCompleteConstants.php

```diff
@@ -0,0 +1,118 @@
+<?php
+namespace GCWorld\FormConfig\Core;
```

Comment:
Move this to Interfaces.  It's 5 folders down. :P

Reply:
Moved to interfaces!